chapters/en/slides/module2/module2_01.Rmd

```diff
@@ -0,0 +1,234 @@
+---
+params:
+  dynamictitle: "module2_01"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+# Effective Use of Visual Channels
+
+Notes:
+So far we have seen how to use points and lines to represent data visually.
+In this lecture,
+we will see how we can also use areas and bars for this purpose.
+Before we dive in to the code,
+let's discuss how different visual channels,
+such as position, area, etc,
+can impact how easy it is for us
+to accurately interpret the plotted data.
+
+As in many cases,
+an efficient way to learn this topic is through personal experience.
+Please follow along in the exercises on the next two slides
+to see which visual channels you find the most effective.
+
+---
+
+## Comparing sizes is easier for some geometrical objects than for others
+
+![](/module2/circle-comparison.png)
+
+Notes:
+In the image in this slide,
+try to estimate how many times larger the big circle is compared to the small one.
+
+---
+
+## Comparing sizes is easier for some geometrical objects than for others
+
+![](/module2/bar-comparison.png)
+
+Notes:
+In the image in this slide,
+try to estimate how many times larger the big bar is compared to the small one.
+
+---
+
+## Both the circles and rectangles differ by 7x
+
+![](/module2/7x.png){width=60%}
+
+Notes:
+In both cases the answer is seven times bigger.
+Even if you guessed both of these exactly correct,
+most people find it is much easier to compare the length or position of the bars
+rather than the area of the circles.
+For the circles,
+you might even have hesitated at exactly what to compare
+when we said "how many time *larger*",
+were you supposed to compare the area or the diameter?
+This is less ambiguous for bars as long as their widths are kept the same.
+
+This is important to keep in mind,
+especially when communicating to others via visualization,
+but also when creating plots for yourself.
+These two examples are originally from [Jeffrey Heer's PyData talk](https://www.youtube.com/watch?v=hsfWtPH2kDg),
+who is visualization researcher at the university of Washington
+and whose research group created D3 and VegaLite
+(which Altair is based on).
+
+---
+
+## Summary of visual channel efficiency
+
```

Comment:
This slide should have a header/footer clearly stating that these numbers have a meaning, and what that meaning is (ranking of ease for humans to distinguish). Some folks may just read the slides...